### DISCUSSION TITLE
How to implement 3ds (strong customer authentication) for card payment using react and react-square-web-payments-sdk #90

### DISCUSSION THREAD

**Discussion by damiano216** [weareseeed/react-square-web-payments-sdk#91] [General]
Hi,
I am working on a project that requires me to implement Strong Customer Authentication (SCA) for online card payments using React and react-square-web-payments-sdk. However, I am facing some issues and I require assistance in this matter.

Could you please provide me with some guidance on how to implement 3DS (Strong Customer Authentication) for card payments using React and react-square-web-payments-sdk? Additionally, if there are any code snippets or examples that you can share with me, it would be greatly appreciated.

Thank you for your help.

**Comment by AndrewBordin** [comment]
Also looking for a solution with this. Every SCA test card that I add returns "userChallenged: false". None of the cards open the 3DS Modal.
